### ISSUE TITLE
Support /whois

### ISSUE THREAD

**Issue by mcepl** [communi/libcommuni#48]
For day-to-day use of IRC I just cannot survive without function ``/whois`` command.

**Comment by mcepl** [comment]
When asking for info on a contact in IRC channell-transported-via-spectrum-to-MUC I got this:
```
[root@muentzer ~]# fg
tail -f /var/log/spectrum2/freenode.cepl.eu/{backends/*,spec*}
2015-12-11 09:30:17,454 INFO  NetworkPluginServer: PING to 0xc8ed60 (ID=11960)
2015-12-11 09:30:18,333 INFO  Component.RAW: RAW DATA IN <iq xmlns='jabber:client' type='get' id='purpleb4548f34' to='#kaprova@freenode.cepl.eu/_petr' from='mcepl@cepl.eu/e38d1b3486d9ba1f3c573bc454973c6dd85c5b0a'><vCard xmlns='vcard-temp'/></iq>
2015-12-11 09:30:18,333 INFO  VCardResponder: mcepl@cepl.eu: Requested VCard of #kaprova
2015-12-11 09:30:18,615 INFO  Component.RAW: RAW DATA IN <iq xmlns='jabber:client' type='get' id='purpleb4548f35' to='#kaprova@freenode.cepl.eu/_petr' from='mcepl@cepl.eu/e38d1b3486d9ba1f3c573bc454973c6dd85c5b0a'><query xmlns='jabber:iq:version'/></iq><iq xmlns='jabber:client' type='get' id='purpleb4548f36' to='#kaprova@freenode.cepl.eu/_petr' from='mcepl@cepl.eu/e38d1b3486d9ba1f3c573bc454973c6dd85c5b0a'><query xmlns='jabber:iq:last'/></iq>
2015-12-11 09:30:18,616 INFO  Component.RAW: RAW DATA OUT <iq from="freenode.cepl.eu" id="purpleb4548f35" to="mcepl@cepl.eu/e38d1b3486d9ba1f3c573bc454973c6dd85c5b0a" type="result"><query xmlns="jabber:iq:version"><name>Spectrum</name><version>2.0.0-alpha</version></query></iq>
2015-12-11 09:30:18,616 INFO  Component.RAW: RAW DATA OUT <iq from="freenode.cepl.eu" id="purpleb4548f36" to="mcepl@cepl.eu/e38d1b3486d9ba1f3c573bc454973c6dd85c5b0a" type="error"><error type="cancel"><feature-not-implemented xmlns="urn:ietf:params:xml:ns:xmpp-stanzas"/></error></iq>

==> /var/log/spectrum2/freenode.cepl.eu/backends/backend-1.log <==
11960: 2015-12-11 09:30:23,459 INFO  IRCConnection: The time has come. Asking /who #kaprova again to get current away states.
11960: 2015-12-11 09:30:23,542 INFO  IRCConnection: End of /who request #kaprova
11960: 2015-12-11 09:30:28,460 INFO  IRCConnection: The time has come. Asking /who #cryptography again to get current away states.
11960: 2015-12-11 09:30:28,543 INFO  IRCConnection: End of /who request #cryptography

==> /var/log/spectrum2/freenode.cepl.eu/spectrum2.log <==
2015-12-11 09:30:37,455 INFO  NetworkPluginServer: PING to 0xc8ed60 (ID=11960)
```
Sounds like something is more fundamentally broken with this (I haven’t asked for ``<status/>`` of whole channel, just one person in it).

**Comment by mcepl** [comment]
``/whois`` (and equivalent ``.whois`` for IM clients which swallow ``/whois`` like Pidgin) has been implemented in hanzz/spectrum2@ab4740c7b6 and hanzz/spectrum2@b44919ed9b .

Thank you.
